src/components/Navigation.js

```diff
@@ -2,10 +2,30 @@ import React from "react";
 import { Link } from "react-router-dom";
 import Logo from "../HotBeansLogo.svg";
 import { ProductConsumer } from "./Context";
-
+import { CustomerConsumer} from "./CustomerContext"
 export default function Navbar() {
   return (
     <nav className="navbar navbar-expand-lg navbar-light bg-light">
+      <div className="mnb navbar navbar-default navbar-fixed-top">
+        <div className="container-fluid">
+          <div className="navbar-header">
+            <button
+              type="button"
+              className="navbar-toggle collapsed"
+              data-toggle="collapse"
+              data-target="#navbar"
+              aria-expanded="false"
+              aria-controls="navbar"
+              onClick={() => {
+                document.body.classList.toggle("msb-x");
```

Comment:
this variable must be part of some local React state,
then you render that `msb-x` element like this:
`<your-element className={classNames({ 'msb-x': this.state.myFlagIsEnabled })}>` (
this lib is super-useful (but not necessary to achieve it): https://github.com/JedWatson/classnames
you could do a simple `if ()` statemement and use className

Reply:
Somebody just made first stateful component